statements/uploaded_document.md

```diff
@@ -0,0 +1,55 @@
+# Uploaded a document
+
+## Description
+
+This event is emitted when a user has uploaded a file of `document` type.
+
+## Example
+
+```json
+{
+   "actor": {
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "https://w3id.org/xapi/netc/verbs/uploaded"
+   },
+   "object": {
+      "id": "http://gaiax.org/xapi/activities/ba297687-b1aa-4477-9efd-a782c8fdb90a",
+      "definition": {
+         "type": "http://id.tincanapi.com/activitytype/document",
+         "name": {
+            "en": "xAPI 101 Introduction notes"
+         }
+      }
+   },
+   "context": {
```

Comment:
Add number of page/sheet ? For example, to get to know if people tend to download if the document is long

Reply:
Do we generally have this information? My guess would be that it is rare to have it